components/ActivityIndicator/README.md

```diff
@@ -86,88 +69,157 @@ import MaterialComponents
-MDCActivityIndicator *activityIndicator =
-    [[MDCActivityIndicator alloc] initWithFrame:CGRectMake(0, 0, 32, 32)];
+MDCActivityIndicator *activityIndicator = [[MDCActivityIndicator alloc] init];
+[activityIndicator sizeToFit];
 [view addSubview:activityIndicator];
 
 // Start animation
 [activityIndicator startAnimating];
-...
+
 // Stop animation
 [activityIndicator stopAnimating];
 ```
 <!--</div>-->
 
-### Determinate indicators
-When indicators are determinate they indicate how long an operation will take when the percentage complete is detectable. The indicator mode must be set to determinate and a progress amount must be provided as a float in the range [0,1].
+
+### Typical use: Determinate
+
+MDCActivityIndicator instances can be shown as determinate by modifying the `indicatorMode`
+property and setting a percentage progress with `progress`. `progress` must be set to a floating
+point number between 0 and 1. Values beyond this range will be capped within the range.
+
+Note: Activity indicators are hidden unless they are animating, even if the indicator is determinate
+progress.
 
 <!--<div class="material-code-render" markdown="1">-->
 #### Swift
 ``` swift
-let activityIndicator = MDCActivityIndicator(frame: CGRect(x: 0, y: 0, width: 32, height: 32))
+let activityIndicator = MDCActivityIndicator()
+activityIndicator.sizeToFit()
 activityIndicator.indicatorMode = .determinate
 activityIndicator.progress = 0.5
 view.addSubview(activityIndicator)
 
-// Start animation
+// Show the indicator
 activityIndicator.startAnimating()
-...
-// Stop animation
+
+// Hide the indicator
 activityIndicator.stopAnimating()
 ```
 
 #### Objective-C
 
 ``` objc
-MDCActivityIndicator *activityIndicator =
-    [[MDCActivityIndicator alloc] initWithFrame:CGRectMake(0, 0, 32, 32)];
+MDCActivityIndicator *activityIndicator = [[MDCActivityIndicator alloc] init];
+[activityIndicator sizeToFit];
 activityIndicator.indicatorMode = MDCActivityIndicatorModeDeterminate;
 activityIndicator.progress = 0.5;
 [view addSubview:activityIndicator];
 
-// Start animation
+// Show the indicator
 [activityIndicator startAnimating];
-...
-// Stop animation
+
+// Hide the indicator
 [activityIndicator stopAnimating];
 ```
 <!--</div>-->
 
-- - -
+### How to theme an activity indicator
 
-## Related Components
+You must first add the Color Themer extension to your project:
 
-<ul class="icon-list">
-  <li class="icon-list-item icon-list-item--components"><a href="../ProgressView">Progress Views</a></li>
-</ul>
+``` bash
+pod 'MaterialComponents/ActivityIndicator+Extensions/ColorThemer'
+```
+
+You can then import the theming APIs:
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+``` swift
+import MaterialComponents.MaterialActivityIndicator_ColorThemer
+```
+
+#### Objective-C
+
+``` objc
+#import "MaterialActivityIndicator+ColorThemer.h"
+```
+<!--</div>-->
+
+MDCActivityIndicatorColorThemer allows you to theme an activity indicator with your app's color
```

Comment:
Consider moving this up to the introductory paragraph. We should explain *why* before telling them *how* to do something.  Consider adding a link to `MDCSemanticColorScheming`.

Reply:
Done. We'll add links to the central theming doc once it's clear where it will live.